docs/.vitepress/config.ts

```diff
@@ -0,0 +1,51 @@
+import { defineConfig } from 'vitepress'
+import { resolve } from 'path'
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+  title: ' ',
+  description: "Remsfal Dokumentation",
```

Comment:
Please writre REMSFAL upper case.

```suggestion
  description: "REMSFAL Dokumentation",
```